src/app/BufferedReadCallback.h

```diff
@@ -74,6 +74,7 @@ class BufferedReadCallback : public ReadClient::Callback
     void OnError(CHIP_ERROR aError) override
```

Comment:
Somewhere in here OnEventData should also do nothing if we have errored, no?

Reply:
sorry, I forgot to push latest change, could you re-review?

Reply:
I see no changes to OnEventData in what's here right now, 5 hours later...

Reply:
yes, onEventData needs to be skipped as well when there is the internal error, thanks